app/assets/javascripts/discourse/app/widgets/post-stream.js

```diff
@@ -261,9 +304,9 @@ export default createWidget("post-stream", {
           result.push(
             new RenderGlimmer(
               this,
-              "div.time-gap.small-action",
+              "div.time-gap.small-action.glimmer-post-stream",
```

Comment:
Why do we need the new class? If we add it, then people might start to depend on it and we'll never be able to remove in future.

If we do need to vary styling, then perhaps we could do that by adding a `.legacy-post-stream` to the old post stream? Then we'll be able to safely remove that class along with the old implementation

Reply:
We don't. This class is mostly there to make things easier for me when developing.

I can instantly tell if I'm looking to a legacy or new component based on the background. I'll have it removed before merging.

Reply:
I removed the `.glimmer_post_stream` class